pkg/controller/podautoscaler/horizontal_test.go

```diff
@@ -360,4 +366,64 @@ func TestEventNotCreated(t *testing.T) {
 	tc.runTest(t)
 }
 
-// TODO: add more tests
+// TestComputedToleranceAlgImplementation is a regression test which
+// back-calculates a minimal percentage for downscaling based on a small percentage
+// increase in pod utilization which is calibrated against the tolerance value.
+func TestComputedToleranceAlgImplementation(t *testing.T) {
```

Comment:
Okay, to be clear,  you are suggesting we add a case **confirming that HPA does not rescale** ?